addons/website/views/snippets/s_carousel.xml

```diff
@@ -47,7 +47,7 @@
                     <div class="container oe_unremovable">
                         <div class="row">
                             <div class="carousel-content col-lg-6 offset-lg-6">
-                                <h2><font style="font-size: 62px; background-color: rgb(255, 255, 255);">Edit this title</font></h2>
+                                <h2><font class="display-3" style="background-color: rgb(255, 255, 255);">Edit this title</font></h2>
```

Comment:
This is wrong though: there could be an inner `font` element with the `display-3-fs` class but not `display-3`. Doing this you would combine the line-height and margins in a weird way no?